scripts/bootstrap-prefix.sh

```diff
@@ -2221,11 +2286,37 @@ bootstrap_stage3() {
 		ln -s bash "${ROOT}"/bin/sh
 	fi
 
-	# Start using apps from the final destination Prefix
-	export PREROOTPATH="${ROOT}/usr/bin:${ROOT}/bin"
```

Comment:
this var used to be picked up by portage, are we sure the below replacement is picked up by portage?